charts/opentelemetry-collector/templates/_config.tpl

```diff
@@ -127,9 +127,6 @@ Build config file for daemonset OpenTelemetry Collector
 {{- if .Values.presets.profilesCollection.enabled }}
 {{- $config = (include "opentelemetry-collector.applyProfilesConfig" (dict "Values" $data "config" $config) | fromYaml) }}
 {{- end }}
-{{- if .Values.presets.coralogixExporter.enabled }}
-{{- $config = (include "opentelemetry-collector.applyCoralogixExporterConfig" (dict "Values" $data "config" $config) | fromYaml) }}
-{{- end }}
```

Comment:
This preset is already processed a few lines above and it cannot be placed after the `headSampling` preset.

Reply:
That's why it's important for us to have examples covering important things. I also believe we should indicate which example is covering a certain PR in its description, so that we can easily review the config rendered in that example. I did it in this PR, for example.